File: src/test/samples/validSamples/validSample.erl
%   #region FirstRegion
X = 42.
   %%         #endregion

% #region Second Region
-module(my_class).
-export([my_method/0]).

%#region  InnerRegion  
my_method() -> ok.
  %%   #endregion ends InnerRegion  

% #region
method_in_unnamed_region() -> ok.
%%%#endregion

%#endregion

File: src/test/samples/validSamples/validSample.fs
//#region   FirstRegion
let x = 42
//#endregion

// #region Second Region
type MyClass() =
    // #region InnerRegion
    member this.MyMethod() = ()
    // #endregion ends InnerRegion

    // #region
    member this.MyOtherMethod() = ()
    //        #endregion
// #endregion

#region NativeRegion
type NativeClass() =
    #region InnerNativeRegion
    member this.NativeMethod() = ()
    #endregion
#endregion

